src/sass/layout/modals/_location.scss

```diff
@@ -0,0 +1,170 @@
+.backdrop {
+  position: fixed;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  background-color: rgba(0, 0, 0, 0.2);
+  opacity: 1;
+  transition: opacity 250ms cubic-bezier(0.4, 0, 0.2, 1),
+    visibility 250ms cubic-bezier(0.4, 0, 0.2, 1);
+}
+
+.is-hidden {
+  opacity: 0;
+  pointer-events: none;
+  visibility: hidden;
+}
+
+.location__modal {
+  position: absolute;
+  top: 55%;
+  left: 50%;
+  transform: translate(-50%, -50%) scale(1);
+  padding: 12px 28px 28px;
+  border-radius: 10px;
+  border: 2px solid #f8d24e;
+  background-color: getColor('white-contacts');
+  transition: transform 250ms cubic-bezier(0.4, 0, 0.2, 1);
+
+  .is-hidden & {
+    transform: translate(-50%, -50%) scale(0);
+  }
+
+  @media screen and (max-width: 767px) {
+    width: 280px;
+    height: 365px;
+  }
+
+  @include tablet {
+    width: 568px;
+    height: 485px;
+    padding: 12px 60px 32px;
+  }
+}
+
+.location__button {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  padding: 0;
+  margin-left: 225px;
+  background-color: inherit;
+  border: none;
+  color: getColor('theme-black');
+  cursor: pointer;
+  width: 20px;
+  height: 20px;
+
+  transition: color 250ms cubic-bezier(0.4, 0, 0.2, 1);
+
+  &:hover,
+  &:focus {
+    color: getColor('orange-main');
+  }
+
+  & .location__icon {
+    fill: currentColor;
+    width: 11px;
+    height: 11px;
+  }
+
+  @include tablet {
+    margin-left: 470px;
```

Comment:
the same as mentioned above.
top: 12px;
left: 12px;

Reply:
margin-left: 470px; - delete
add:
left: 94%;
    top: 12px;